iset.mm

```diff
@@ -71022,6 +71026,34 @@ Similar to the Axiom of Choice (first form) of [Enderton] p. 49.
       DZUJUIDZEZBFGZAFGZHIHIUOIUIUJJZABKZUJUIJZLZBFGZAFGUOABMUTUNAFUIFJZUSUMBFV
       AUJFJZQZUPUMUQURVCUPUKUMVBUPUKRVAUJUINOUKULUAZPUQUMRVCUQUKUMUIUJUBVDSUCVC
       URULUMVAURULRVBUIUJNUDULUKUEPUFUGUHSTT $.
+
+    $( Double negated ordinal trichotomy.  (Contributed by Jim Kingdon,
+       25-Aug-2024.) $)
+    nontri3or $p |- ( -. -. EXMID
+        <-> -. -. A. x e. On A. y e. On ( x e. y \/ x = y \/ y e. x ) ) $=
+      ( wem wn wel weq w3o con0 wral onntri51 onntri13 onntri35 syl impbii ) CD
+      DZABEABFBAEGZBHIAHIDDZABJQPDDBHIAHIOABKABLMN $.
+
+    $( Double negated ordinal trichotomy.  (Contributed by Jim Kingdon,
+       25-Aug-2024.) $)
+    nontri2or $p |- ( -. -. EXMID
+        <-> -. -. A. x e. On A. y e. On ( x C_ y \/ y C_ x ) ) $=
```

Comment:
Can you add to these two comments whether the biconditional `-. EXMID <-> -. A. x ...` holds, or if only the reverse implication holds, and if the biconditional holds, same question when removing one more negation.

I'm asking that because when we see this theorem, that's really the first question that comes to (my) mind, and we are (at least, I am) really curious to know the answer. Yes, they may be easy and/or already in some comments a few lines away, but for the reader's convenience, having the remark here really helps.

Reply:
Do you know the answer? Or, perhaps more subtlely, whether that is a question with larger implications or not?

This all came up in the context of https://mathoverflow.net/questions/452678/does-negative-trichotomy-hold-for-constructive-ordinals which concerns the doubly negated form, which is why work to date has mostly been on that (by James E. Hanson as much as me, to be honest, I just formalized his mathoverflow comment)

Reply:
I do not know. I have not thought much about it either. If this is unknown (to us, or to everyone), we can also be open about that in the comment. I don't know about larger implications, but these questions are very natural, given the form of the statements.

Obviously, `|- ( EXMID -> tricho )` (so by intuitionistic contrapositive, we have `|- ( -. tricho -> -. EXMID )`, etc.).

So, questions (and I haven't thought about them, maybe it's easy, maybe not):
* `|-?- ( -. EXMID -> -. tricho )`, and if yes,
* `|-?- ( tricho -> EXMID )`.

Reply:
OK, easiest way to handle this, I guess, is to just to do some work that I had been planning to do later.

The pull request now includes ` EXMID <-> A. x e. On A. y e. On ( x e. y \/ x = y \/ y e. x ) ` and ` EXMID <-> A. x e. On A. y e. On ( x C_ y \/ y C_ x ) ` .

Reply:
These two results are much nicer !  I'm happy I asked.

Reply:
> These two results are much nicer ! I'm happy I asked.

I don't always act on suggestions from other people, but in this case, I did and I'm glad to hear it turned out well!

(Well and glad that it turned out to be fairly easy - the only real difficulties had to do with things like names and what should or should not be broken out into a separate theorem).